src/modules/imageresizer/tests/ImageResizerUITest.csproj

```diff
@@ -1,114 +1,44 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\..\Version.props" />
-  <!-- We don't have GenerateAssemblyInfo task until we use .net core, so we generate it with WriteLinesToFile -->
+  
   <PropertyGroup>
-    <AssemblyTitle>ImageResizerUITest</AssemblyTitle>
-    <AssemblyCompany>Microsoft Corp.</AssemblyCompany>
-    <AssemblyCopyright>Copyright (C) 2020 Microsoft Corp.</AssemblyCopyright>
-  </PropertyGroup>
-  <ItemGroup>
-    <AssemblyVersionFiles Include="Generated Files\AssemblyInfo.cs" />
-  </ItemGroup>

```

Comment:
I had to delete the AssemblyInfo.cs manually for this project and the ImageResizerUI project as .net core, as this was already generated from a previous build and .net core projects add the files by default.  This was causing [cs0579](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0579).  I might be a good idea to add this to the description before closing the PR, just in case people are confused about compile errors on their local box after fetching the latest.